script_library/pre-flight-checks.sh

```diff
@@ -56,6 +56,15 @@ check_caasp4_terraform_available(){
         exit
     fi
 }
+
+check_caasp4_terraform_version(){
+  echo "Checking for terraform version (should be lower than 0.12)"
+  ${TERRAFORM_BINARY_PATH} version|egrep -qv "^Terraform\sv0\.12\.[0-9]{2}$"
```

Comment:
Can you check for 0.11 so we are good even if there is a new (eg. 0.13) terraform available?

Reply:
good point